packages/api/utils/validation.test.ts

```diff
@@ -206,8 +167,8 @@ describe("isApplicationStep()", () => {
   it("should accept an ApplicationStep object", async () => {
     expect(isApplicationStep(mockApplicationStep)).toEqual(true);
   });
-  it("should reject an ApplicationStep missing any field", async () => {
-    expect(isApplicationStep(mockApplicationStepsMissingFields)).toEqual(false);
+  it.each(mockApplicationStepsMissingFields)("should reject an ApplicationStep missing any field", async (item) => {
```

Comment:
Love to see more usage of `each`

Reply:
This test was falsely passing because the array wasn't an `ApplicationStep`.  Fixed this instance and a few others that were similar.

Reply:
This, among other changes in the PR, increased our test count to 280.